stable/selenium/templates/chrome-deployment.yaml

```diff
@@ -14,7 +14,10 @@ spec:
         chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
         release: "{{ .Release.Name }}"
         heritage: "{{ .Release.Service }}"
-      annotations: 
+      {{- if .Values.chrome.podAnnotations }}
+      annotations:
+{{ toYaml .Values.chrome.podAnnotations | indent 4 }}
```

Comment:
the indenting seems off for all of them?

Reply:
:/ you are correct. Fixing now